Cargo.toml

```diff
@@ -12,3 +12,7 @@ pretty_env_logger = "0.5.0"
 regex = "1.10.6"
 serde = { version = "1.0", features = ["derive"] }
 serde_yaml_ng = "0.10.0"
+
+[lib]
+name = "config"
+path = "src/lib.rs"
```

Comment:
Perhaps we don't need this section because we're not going to publish this crate as a library?

```suggestion
```